app/models/async_transaction/base.rb

```diff
@@ -3,6 +3,10 @@
 module AsyncTransaction
   class Base < ActiveRecord::Base
     self.table_name = 'async_transactions'
+
+    INITIAL_STATUS = 'requested'
+    END_STATUS = 'completed'
```

Comment:
In our Vet360 service itself we've named our constants to reflect the actual value, so in this case `REQUESTED` and `COMPLETED`, but I definitely see the reasoning here for the naming choices (the predicate for checking status reads a lot better with these). What do you think @hpjaj?

Reply:
I'm not a fan of naming constants after their value rather than their purpose. I'll make this change but this feels much less clear to a future developer.